File: TheBookLore/app/src/main/java/com/penguin/thebooklore/viewmodel/HomeViewModel.java
package com.penguin.thebooklore.viewmodel;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HomeViewModel extends ViewModel {

    private MutableLiveData<String> mText = new MutableLiveData<>();

    public MutableLiveData<String> getText() {
        return mText;
    }

    public HomeViewModel() {
        mText.setValue("This is home fragment");
    }

}

File: TheBookLore/app/src/main/java/com/penguin/thebooklore/viewmodel/HomeViewModel.kt
package com.penguin.thebooklore.viewmodel

import android.app.Application
import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class HomeViewModel(application: Application) : BaseViewModel(application) {

    val text = MutableLiveData<String>()

    init {
        text.value = "This is home fragment"
    }

}